src/pointer-analysis/value_set.cpp

```diff
@@ -51,6 +51,13 @@ bool value_sett::field_sensitive(
   return ns.follow(type).id()==ID_struct;
 }
 
+const value_sett::entryt *value_sett::find_entry(const value_sett::idt &id)
```

Comment:
Could we use `optionalt` instead of the pointer?

Reply:
I'd want optionalt<reference_wrapper<entryt>>, which is I feel just a longhand way of writing a pointer. See also `symbol_tablet::lookup` (returns raw pointer) and so on.